js/src/features/human-resouce/components/HumanResourceCalendar/HumanResourceCalendar.tsx

```diff
@@ -1,72 +1,117 @@
 import React, { useReducer, useRef } from "react";
 import { LeftOutlined, RightOutlined } from "@ant-design/icons";
 import formatDate from "date-fns/fp/format";
-import { EmployeeShort, AbsencesMap } from "HumanResourceModels";
+
+import { EmployeeShort, AbsencesMap, VacationMap } from "HumanResourceModels";
 import { HumanResourceCalendarDay } from "./HumanResourceCalendarDay";
 import { HumanResourceCalendarUser } from "./HumanResourceCalendarUser";
 import { initial, reducer } from "./reducer";
-import { getDateInterval, getMonthsFromInterval } from "./helpers";
+import {
+  getAbsencesDays,
+  getDateInterval,
+  getMonthsFromInterval,
+} from "./helpers";
 
 import "./HumanResourceCalendar.scss";
+import { HumanResourceCalendarUserAvatar } from "./HumanResourceCalendarUserAvatar";
```

Comment:
this import should be above the css import

Reply:
Fixed